packages/wallet/src.ts/index.ts

```diff
@@ -222,6 +321,80 @@ export class Wallet extends Signer implements ExternallyOwnedAccount, TypedDataS
 		return encryptKeystore(this, password, options, progressCallback);
 	}
 
+	/**
+	 * Performs a contract local call (ContractCallQuery) against the given contract in the provider's network.
+	 * In the future, this method should automatically perform getCost and apply the results for gasLimit/txFee.
+	 * TODO: utilize getCost when implemented
+	 *
+	 * @param txRequest - the call request to be submitted
+	 */
+	async call(txRequest : Deferrable<TransactionRequest>): Promise<string> {
```

Comment:
Shouldn't we place this logic inside the `signer` in order to have the same interface as `ethers`?

Reply:
we cannot as we can't access `this.account` and `this._signingKey()` in `abstract-signer`

Reply:
if we move it to signer we won't have access to this.account

Reply:
I've moved the logic but kept the `getAddress` promise